### DISCUSSION TITLE
Stop posting here

### DISCUSSION THREAD

**Discussion by frogmasamune** [3kh0/ext-remover#631] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
There are a bunch of sysadmins lurking here, like u/vawlk. By posting exploits here you just alert them, then they post the fixes on k12sysadmin and we all get screwed. This applys especially to 3kh0 (like did you wait 10 minutes before posting the newest exploit?), 3kh0 just remove all the exploits and hide a link to the discord in there somewhere, that at least makes it harder for sysadmins to find stuff.

**Comment by obkduishinijf** [comment]
if one finds it then they all have it, genius

**Comment by bartholemuew** [comment]
they are nerds lmao!!!

**Comment by piplup7575** [comment]
damn thats crazy but who asked
this is just a repo of skids

**Comment by 3kh0** [comment]
They are already in TN and more so it is unless to hide like a coward

**Reply by S-PScripts** [reply]
blud said unless 💀
